### DISCUSSION TITLE
User management

### DISCUSSION THREAD

**Discussion by jpprost** [consuldemocracy/consuldemocracy#5659] [Q&A] [answered]
Hello,

I fail to understand how to manage the users: how to access basic CRUD functionalities, how to assign them roles, grant them a public official level, etc. 

For information, I disabled the verification process.

The documentation does not seem to cover the topic. Also, the community has been unreachable for quite a while.

Thank you for your help.

Cheers,
JP

**Comment by javierm** [comment, marked as answer]
Hi, @jpprost :smile:.

I'm very sorry for the late answer! :pray: Somehow we missed this one back in August.

You can manage roles using the links under "profiles" in the admin area. The interface is confusing, though; see https://github.com/consuldemocracy/consuldemocracy/issues/3950#issuecomment-742428393 for information about how to do it.

There is no CRUD functionality for users themselves; administrators cannot create or edit other users. However, it's possible to block users by going to the moderation area (under `/moderation`) and clicking on "Block users".

Does this cover what you need?

**Reply by jpprost** [reply]
Hi @javierm!
yes, thank you, it does help. The important thing to know is that you need to search the user you want to assign a role to. That makes it a bit tedious, but that's ok.
As for the CRUD op on the users, I take that this (lack of) feature is to ensure that even the site admins can not manipulate the participative data beyond basic moderation. It makes sense.

Cheers,
JP

**Reply by javierm** [reply]
> As for the CRUD op on the users, I take that this (lack of) feature is to ensure that even the site admins can not manipulate the participative data beyond basic moderation.

That's correct. But I forgot to mention that it is indeed possible to add users via the management area. The purpose of this area is to provide a way for citizens without internet connection to participate by asking a Consul Democracy manager/admin to sign them up. In order to use this feature, you'll have to either configure a connection to a remote census (using the census settings in the administration area) or add records to the local census (going to "Settings -> Local census" in the administration area).
